### ISSUE TITLE
Truncated or partly delivered message

### ISSUE THREAD

**Issue by kaktusztea** [signalapp/Signal-iOS#5924]
- [x] I have searched open and closed issues for duplicates
- [x] I am submitting a bug report for existing functionality that does not work as intended
- [x] This isn't a feature request or a discussion topic

Alice: xxx873⁩
Bob: xxx759

There is some very strange message behaviour in a Signal group.
Group members: 6

1. First message sent by Alice: 1733916758796
2. Alice edited the message
3. Bob (receiver group member) got the message in a truncated form. Some other receiver in the group got it right...
4. Bob (receiver group member) deletes this very message in his desktop client. It is being deleted on all linked devices.

... a few hours later ...

5. Second message sent by Alice: 1733934206482  (same sender)
6. Bob (receiver group member) iOS device DID NOT receive this message!
7. Bob (receiver group member) Desktop device DID receive this message (same receiver user!)

Android sender
https://debuglogs.org/android/7.26.1/5bea5e7300b6a27716bbaabf8e6577e86347b55cb96d53f54a0e00bcf213c777

iOS receiver - second message not appeared
https://debuglogs.org/ios/7.39.0/9ff2bc143e3f46302403e8df2eb0481f3a5920fd4f1aff85015f3275cf69415a.zip

Desktop receiver
https://debuglogs.org/desktop/7.35.0/76e3ba11aa985b03de539c91aab2aced0fd126e4872c27165d19b39ae9d3bd85.gz

**Comment by kaktusztea** [comment]
Please reply if you checked on this.

**Comment by kaktusztea** [comment]
Please reply if you checked on this, then I can close it.

**Comment by max-signal** [comment]
@kaktusztea have you seen this issue repeatedly? Or just once? The latest release (v7.72) has some tweaks related to edited messages, and I suspect it might cause Desktop & iOS to behave more consistently when following these steps.

**Comment by kaktusztea** [comment]
No, I haven't seen this in the recent year, just waited for response.
Now I close it, will open a new one if I face this again.
